librad/src/meta/entity.rs

```diff
@@ -647,6 +619,7 @@ where
 
             // End at root revision
             if revision == 1 {
+                self.status = VerificationStatus::Verified;
```

Comment:
So I set the `status` here. But in the test suite, it comes out as `Unknown` :thinking: @massimiliano-mantione: maybe you have an idea of what's going on.

Reply:
Ah it's because of the `clone` here: https://github.com/radicle-dev/radicle-link/pull/126/files#diff-e8128eef3fbc5e6398034e0c89623737R113 :facepalm: